client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -188,6 +196,15 @@ private void logFailedRequest(RequestBuilder fullReq, Throwable throwable) {
             .flatMap(str -> deserialize(method, str));
     }
 
+    private Observable<? extends ByteBuf> logEmptyResponse(RequestBuilder request, HttpClientResponse<ByteBuf> response) {
+        return defer(() -> {
+
```

Comment:
can we use the loggers built in supplant function instead?

Reply:
Since  I need to return an observable here I think not